balancer/grpclb/grpclb_picker.go

```diff
@@ -49,6 +49,14 @@ func newRPCStats() *rpcStats {
 	}
 }
 
+func isZeroStats(stats *lbpb.ClientStats) bool {
+	return len(stats.CallsFinishedWithDrop) == 0 &&
```

Comment:
All fields need to be accessed atomically

Reply:
These are in the stats _message_, not in the rpcStats struct.